lib/vrl/stdlib/src/string.rs

```diff
@@ -0,0 +1,45 @@
+use vrl::prelude::*;
+
+#[derive(Clone, Copy, Debug)]
+pub struct STring;
```

Comment:
I'd rather disambiguate by the namespaces than resorting to this spelling.

Reply:
Yeah, not sure actually why I did this. Doesn't seem to be a need for it, really.